localisation/00_zuwc_united_localisation.csv

```diff
@@ -3019,20 +3019,18 @@ wonder_wall_northfold_name_string;Northfold Thoradin's Wall;;;;;;;;;;;;;x
 wonder_wall_thoradin_desc_string;Thoradin's Wall, an immense edifice, was once a symbol of humanity's strength and defensive might.;;;;;;;;;;;;;x
 wonder_amphitheater_surohi_name_string;Great Arena;;;;;;;;;;;;;x
 wonder_amphitheater_surohi_desc_string;The ancient Gurubashi trolls built a huge gladiatorial arena many years ago.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+wonder_aztec_pyramid_nazwatha_name_string;Atul'nazman;;;;;;;;;;;;;x
+wonder_aztec_pyramid_necropolis_name_string;Necropolis;;;;;;;;;;;;;x
+wonder_aztec_pyramid_necropolis_desc_string;The spirits of departed can be seen approaching it.;;;;;;;;;;;;;x
+wonder_aztec_pyramid_atulaman_name_string;Atul'Aman;;;;;;;;;;;;;x
+wonder_zuldazar_name_string;Dazar'alor;;;;;;;;;;;;;x
+wonder_zuldazar_desc_string;Dazar'alor is an enormous pyramid with about 11 or 12 levels.;;;;;;;;;;;;;x
+wonder_mausoleum_ataldazar_name_string;Atal'Dazar;;;;;;;;;;;;;x
+wonder_mausoleum_ataldazar_desc_string;For millennia, the Zandalari kings and queens were laid to rest within the tombs of Atal'Dazar. These pyramids grew more elaborate with each passing ruler, as new rooms were constructed to house the deceased's vast wealth.;;;;;;;;;;;;;x
```

Comment:
```suggestion
wonder_mausoleum_ataldazar_desc_string;For millennia, Zandalari kings and queens have been laid to rest within the tombs of Atal'Dazar. These pyramids have grown more elaborate with each passing ruler as new rooms have been constructed to house the deceased's vast wealth.;;;;;;;;;;;;;x
```

Reply:
Weird flex, but ok
https://wow.gamepedia.com/Adventure_Guide_Battle_for_Azeroth_dungeons#Atal.27Dazar

Reply:
Not sure what you mean, but my suggestion still stands, so it's up to you.

Reply:
It's a literally a wow in-game description, but I implemented your suggestion anyway.